daml-lf/transaction/src/test/scala/com/digitalasset/daml/lf/transaction/TransactionSpec.scala

```diff
@@ -275,9 +273,7 @@ class TransactionSpec extends AnyFreeSpec with Matchers with ScalaCheckDrivenPro
 
   "contractKeys" - {
     "return all the contract keys" in {
-      // TODO: https://github.com/digital-asset/daml/issues/8020
-      // change VDev to  TransactionVersion.StableVersions.max once exception are released
-      val builder = TransactionBuilder(TransactionVersion.VDev)
+      val builder = TransactionBuilder(TransactionVersion.minExceptions)
```

Comment:
The test originally works with the latest stable version. 
I don't think we want to change that.

Reply:
Reverted in e9a87bb0741c2bfaae8ca23bd616b22a76d7d56c for now.
What are you suggesting we do here in the long term? We clearly cannot go below minExceptions if we use exceptions. Split up tests in some that use exceptions and some that don’t?